containers-tests/tests/intmap-properties.hs

```diff
@@ -1469,36 +1449,36 @@ prop_deleteMaxModel ys = length ys > 0 ==>
 prop_filter :: Fun Int Bool -> [(Int, Int)] -> Property
 prop_filter p ys = length ys > 0 ==>
   let xs = List.nubBy ((==) `on` fst) ys
-      m  = filter (apply p) (fromList xs)
+      m  = filter (applyFun p) (fromList xs)
```

Comment:
Why this change? `apply` isn't deprecated, and it's shorter.

Reply:
No problem, let's stick to `apply`.